test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua

```diff
@@ -0,0 +1,81 @@
+----------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL must transfer request to HMI in case of "autoCompleteList" param have value is UpperBound
+--
+-- Description:
+-- Case when SDL tranfer SetGlobalProperties_request with <autoCompleteList> param to HMI
+-- respond with <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>  string in array is upper bound.
+-- 3. HMI respond.
+--
+-- Expected result:
+-- SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+----------------------------------------------------------------------------------------
+---[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_AutoCompleteList_String_in_Array_Is_UpperBound()
+--mobile side: sending SetGlobalProperties request
+local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+    {
+      keyboardProperties =
+      {
+        keyboardLayout = "QWERTY",
+        keypressMode = "SINGLE_KEYPRESS",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = {"upperbound", "upperboundupperboundupperboundupperboundupperbound"}
```

Comment:
@Aleksandriys, due to "[Mobile API] SetGlobalProperties request/response" non-functional req, maxlength=50.
You use 51 in script, so, actually, you check **out upper bound** case, which is not applicable.
Please update script to check upper bound value, not out upper bound.